src/components/Comment.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+    Box,
+    Button,
+    Container,
+    Grid,
+    Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+        borderStyle: 'solid'
+    },
+
+    date: {
+        float: 'left',
+        textAlign: 'left',
+        fontSize: '18px',
+        marginTop: '0.25rem',
+    },
+
+    content: {
+        marginTop: '0.75rem'
+    },
+    
+    deleteComment: {
+        paddingLeft: '2rem',
+        paddingRight: '2rem',
+        float: 'right',
+        textAlign: 'right',
+        background: 'lightgrey'
+    }
+
+});
+
+interface CommentProps {
+    username: string;
+    date: string;
+    claps: number;
+    content: string;
+}
+
+const Comment: React.FC<CommentProps> = ({username, date, claps, content}) => {
+    const classes = useStyles();
+
+    return(
```

Comment:
Is this linted? There should be a space here for sure per linting

Reply:
Should we start using [ESLint](https://eslint.org/docs/user-guide/getting-started)?